src/views/Correction/FmCorrection.vue

```diff
@@ -0,0 +1,221 @@
+<template>
```

Comment:
For now, this is mostly a copy of the original Correction.vue page (with a few updates).

More to come in future commits...